site/content/en/docs/Guides/feature-stages.md

```diff
@@ -38,19 +38,6 @@ The current set of `alpha` and `beta` feature gates:
 | [Split `agones-controller` ](https://github.com/googleforgames/agones/issues/2797)                                    | `SplitControllerAndExtensions` | Disabled | `Alpha` | 1.30.0 |
 | Example Gate (not in use)                                                                                             | `Example`                      | Disabled | None    | 0.13.0 |
 {{% /feature %}}
```

Comment:
This feature block should be removed since the enclosed content gets published with this release.

Reply:
removed the feature block in feature-stages.md